content/applications/marketing/marketing_automation/getting_started/target_audience.rst

```diff
@@ -0,0 +1,74 @@
+==================
+Target an audience
+==================
+
+Getting marketing campaigns to the *right* audience is paramount when trying to grow a business.
+Fortunately, the :guilabel:`Odoo Marketing Automation` application provides the right tools to take
+marketing strategies to the next level.
+
+Work with target filters
+========================
+
+When configuring the filters on a marketing campaign, there are some options that have an arrow
+icon beside them, this signifies the parameter has more variables within it.
+
+.. image:: target_audience/marketing-filters.png
+   :align: center
+   :alt: The drop-down filter menu in the Marketing Automation application.
+
+More than one parameter line can be configured. A :guilabel:`node (+)` can be added under what is
+called a :guilabel:`branch (…)`. To add a node, click on the :guilabel:`+` (plus sign icon) and to
+add another branch click on the :guilabel:`...` (ellipses icon).
+
+To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on :guilabel:`Delete node (x)`.
+
+.. image:: target_audience/marketing-filter-nodes.png
+   :align: center
+   :alt: The drop-down filter menu in the Marketing Automation application.
+
+Every time a new branch is created, there are two options, either the records can match
+:guilabel:`all` criteria for the upcoming rules, or the records can match :guilabel:`any` (meaning
+one rule or the other). To change between these two options, simply click the drop-down arrow icon 
+in the green box and select :guilabel:`Any` or :guilabel:`All`.
```

Comment:
Suggestion to rewrite this entire section for more context in the writing and clarity. 
In order:
1. define the terms in hierarchical order (branch > node)
2. then provide usage instructions per term 
3. screenshot follows last (as _supplemental_ content, not the _main_ content that interrupts the writing)

```suggestion
Filters can be extended by adding *branches* and *nodes*. A *branch* creates a new line of
parameters, and further allows parameters to be clustered with *Any* or *All* statements, and a
*node* adds another parameter to the branch. 

Every time a new branch is created, there are two options: 

- either the records can match :guilabel:`ALL` criteria for the upcoming rules (an AND statement
where all criteria must match),
- or the records can match :guilabel:`ANY` criteria for the upcoming rules (an OR statement where
only one of the criteria must match). 

To change between these two options, simply click the drop-down arrow icon in the green box and
select :guilabel:`Any` or :guilabel:`All`.

To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another branch click on the
:guilabel:`... (ellipses icon)`. To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on
:guilabel:`x (delete)`.

.. image:: target_audience/marketing-filter-nodes.png
   :align: center
   :alt: The drop-down filter menu in the Marketing Automation application.
```